src/Entities/DonationPayments/SofortPayment.php

```diff
@@ -4,44 +4,46 @@
 
 namespace WMDE\Fundraising\Entities\DonationPayments;
 
+use DateTime;
 use Doctrine\ORM\Mapping as ORM;
+use WMDE\Fundraising\Entities\DonationPayment;
 
 /**
  * @since 6.0
  *
  * @ORM\Table(name="donation_payment_sofort")
  * @ORM\Entity
  */
-class SofortPayment {
+class SofortPayment extends DonationPayment {
 
 	/**
 	 * @var string
 	 * Example value: W-Q-ABCDEZ
 	 *
 	 * @ORM\Column(name="transfer_code", type="string", length=10, unique=true)
-	 * @ORM\Id
 	 */
 	private $bankTransferCode = '';
 
 	/**
-	 * @var integer
-	 * Example value: 1337
+	 * @var DateTime
 	 *
-	 * @ORM\Column(name="donation_id", type="integer", unique=true)
+	 * @ORM\Column(name="confirmed_at", type="datetime", nullable=true)
 	 */
-	private $donationId = '';
+	private $confirmedAt;
```

Comment:
Is this specific to sofort?

When and how does this get set? I can't tell if this makes sense here or not without knowing a bit more.